src/libcore/ops/drop.rs

```diff
@@ -99,9 +99,15 @@ pub trait Drop {
     /// Given that a [`panic!`] will call `drop` as it unwinds, any [`panic!`]
     /// in a `drop` implementation will likely abort.
     ///
+    /// Note that even if this panics, the value is considered to be dropped;
+    /// you cannot cause it to be called again. This is normally automatically
```

Comment:
```suggestion
    /// you must not cause `drop` to be called again. This is normally automatically
```
"it" could be the value here (makes no sense semantically, but still)